src/storage/index/LookupBaseProcessor.inl

```diff
@@ -316,21 +337,14 @@ LookupBaseProcessor<REQ, RESP>::buildPlanWithData(const cpp2::IndexQueryContext&
         return nullptr;
     }
 
-    auto schemaName = planContext_->isEdge_
-                      ? this->env_->schemaMan_->toEdgeName(spaceId_, planContext_->edgeType_)
-                      : this->env_->schemaMan_->toTagName(spaceId_, planContext_->tagId_);
-    if (!schemaName.ok()) {
-        return nullptr;
-    }
-
```

Comment:
line 312- line 315 Is not needed.
becase line 25 you have handled

Reply:
good catch

Reply:
Fixed